gpkit/constraints/relax.py

```diff
@@ -0,0 +1,140 @@
+"""Models for assessing primal feasibility"""
+from .model import Model
+from .set import ConstraintSet
+from ..nomials import Variable, VectorVariable, parse_subs, NomialArray
+
+
+class RelaxAll(Model):
```

Comment:
After talking yesterday, I think we're agreed that the use case you're thinking of is for now better done manually?